client/src/carousel.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled from 'styled-components';
+
+// === STYLES === //
+
+const Wrapper = styled.div`
+  width: 1280px;
+  overflow: hidden;
+`;
+
+const CarouselContainer = styled.div`
+  display: flex;
+  margin: 0 0 20px 20 px;
+  align-content: center;
+`;
+
+// flex: 0 (flex-grow off) 1 (flex shrink on )
+const CarouselSlot = styled.div`
+  flex: 0 1 100%;
+  flex-basis: 80%;
+  margin-right: 20px;
+  order: ${(props) => props.order};
+`;
+
+const Title = styled.div`
+  font-family: Alternate Gothic No1 D;
+  font-size: 20px;
+  color: #4c4c4c;
+`;
+
+const Role = styled.div`
```

Comment:
I see all your styled components are divs. Is that what your component requires? I used many different elements from spans, ul's etc. Just something to look into.

Reply:
This is a good point. I've been using divs as a default to limit the complexity while trying new things. I'll look into what might be more appropriate when I refactor.